### PR TITLE
Add config to remove CDATA

### PR DESCRIPTION
**Pull request by riccardonar**

### CODE REVIEW

**Comment by goetas** [comment]
instead of having a generic array of configurations, why hot have just `$useCData = true` ?

For now we have no special configurations for PHP.

**Comment by riccardonar** [comment]
I think there may be other useful configurations. So it seemed to me to maintain a certain order

File: src/DependencyInjection/Xsd2PhpExtension.php [hunk 1]
```diff
@@ -50,6 +50,11 @@ public function load(array $configs, ContainerBuilder $container)
             }
         }
 
+        if ($config['configs_jms']) {
```
Comment (goetas) [inline src/DependencyInjection/Xsd2PhpExtension.php:53]:
```suggestion
        if (isset($config['configs_jms'])) {
```

Comment (goetas) [reply src/DependencyInjection/Xsd2PhpExtension.php:53]: otherwise it will never get to have `false`

File: src/Jms/YamlConverter.php [hunk 2]
```diff
@@ -315,6 +324,9 @@ private function handleClassExtension(&$class, &$data, Type $type, $parentName)
             $property["accessor"]["getter"] = "value";
             $property["accessor"]["setter"] = "value";
             $property["type"] = $alias;
+            if (!$this->useCdata){
```
Comment (goetas) [inline src/Jms/YamlConverter.php]: do we need the if condition? cant we just assign it directly?

**Review by goetas — CHANGES_REQUESTED** [review]
see comments

File: src/DependencyInjection/Xsd2PhpExtension.php (hunk 1 above)
Comment (goetas) [reply src/DependencyInjection/Xsd2PhpExtension.php:53]: my bad, it is ok

File: src/Jms/YamlConverter.php (hunk 2 above)
Comment (riccardonar) [reply src/Jms/YamlConverter.php]: true is the default option for useCdata and jms configuration, so i set it only it's is false.
In this way the jms config files are identical before this commit. And also tests not change

File: src/DependencyInjection/Xsd2PhpExtension.php (hunk 1 above)
Comment (riccardonar) [reply src/DependencyInjection/Xsd2PhpExtension.php:53]: configs_jms is an array, always with default

File: src/Jms/YamlConverter.php [hunk 3]
```diff
@@ -420,6 +435,9 @@ private function visitElement(&$class, Schema $schema, ElementItem $element, $ar
         $property["access_type"] = "public_method";
         $property["serialized_name"] = $element->getName();
 
+        if (!$this->useCdata){
```
Comment (goetas) [inline src/Jms/YamlConverter.php]:
```suggestion
        if (!$this->useCdata) {
```

Comment (riccardonar) [reply src/Jms/YamlConverter.php]: ok, fixed

**Comment by goetas** [comment]
Thanks!

**Pull request merged**